### PR TITLE
This adds the online statistics for MS/MA under Live tab

### PR DESCRIPTION
**Pull request by akopich**
During a capture Live tab has been showing the online
statistics only for Dynamically instrumented functions,
while MS and MA were shown only upon capture finalization.
This fixes that.

Tests: Manual, Unit
Bug: http://b/232218601

### CODE REVIEW

**Review by florian-kuebler — CHANGES_REQUESTED** [review]
Thanks for the change, however, I don't think we want to use Qt's event system here. We should simply go with a listener or observer pattern.

File: src/DataViews/LiveFunctionsDataView.cpp [hunk 1]
```diff
@@ -528,9 +528,12 @@ void LiveFunctionsDataView::OnDataChanged() {
 }
 
 void LiveFunctionsDataView::OnTimer() {
-  if (app_->IsCapturing()) {
-    OnSort(sorting_column_, {});
+  if (!app_->IsCapturing()) return;
+  for (uint64_t missing_scope_id : FetchMissingScopeIds()) {
+    indices_.push_back(missing_scope_id);
```
Comment (beckerhe) [inline src/DataViews/LiveFunctionsDataView.cpp:533]: Can you elaborate on the performance implications? Do we get a problem when we run this every 300ms and the total number of scope ids is large?

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/DataViews/LiveFunctionsDataView.cpp (hunk 1 above)
Comment (akopich) [reply src/DataViews/LiveFunctionsDataView.cpp:533]: I've just instrumented 1k functions. The thing is pretty fast. The step I have added takes ~100us. I don't see the reason to bother with trickier structures right now. 

Should I add a comments about that?

**Pull request merged**